iCubGenova09/icub_wbd.xml

```diff
@@ -106,7 +107,7 @@
     <xi:include href="calibrators/torso-calib.xml" />
 
     <!-- estimators -->
-    <xi:include href="estimators/wholebodydynamics.xml" /> 
+    <!--<xi:include href="estimators/wholebodydynamics.xml" />  -->
```

Comment:
Did you tested with ``wholebodydynamics`` by any chance? Is it commented because it was not working?

Just for reference, when doing some tests yesterday, we had to comment it out because the FTs were not running. Did you simply copy the file that was already in the robot head?

Reply:
Not tested, I found it already commented in `devel`

Reply:
Nope, I  just edited the file as I found it

Reply:
Ok clear!